pkg/opensearch/client/search_request.go

```diff
@@ -506,9 +573,36 @@ func (b *SearchRequestBuilder) SetTraceSpansFilters(to, from int64, traceId stri
 
 }
 
+// Stats adds the needed aggregations for the Stats request, used to
+// display latency and throughput
+func (b *aggBuilderImpl) Stats() AggBuilder {
+	b.Terms("service_name", "serviceName", func(a *TermsAggregation, b AggBuilder) {
+		a.Size = 500
```

Comment:
@sarahzinger It's a random number since we don't expect there to be that many services to be returned. But thinking about it now, i think it could be better to increase it to 1000, since some of the AWS folks mentioned users might have a lot of services. I feel like this could also be a query option (suggested as a feature request [here](https://raintank-corp.slack.com/archives/C01LVB0CYTW/p1714766356905789)) in the future